src/plugins/dashboard/public/application/components/dashboard_listing.tsx

```diff
@@ -90,22 +89,26 @@ export const DashboardListing = () => {
   };
 
   const editItem = useCallback(
-    ({ editUrl }: any) => {
```

Comment:
we will likely need to revisit the dashboards providers work as this appears to have broken functionality of that.